.github/workflows/codespaces-test.yml

```
@@ -0,0 +1,73 @@
+name: Test Codespaces Setup
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)